setup.py

```diff
@@ -592,6 +592,16 @@ def make_relative_rpath(path):
 extensions.append(THNN)
 
 if WITH_CUDA:
+    THNVRTC = Extension("torch._nvrtc",
+                        libraries=['nvrtc', 'cuda'],
+                        sources=['torch/csrc/nvrtc.cpp'],
+                        language='c++',
+                        include_dirs=include_dirs,
+                        library_dirs=library_dirs + [cuda_lib_path + '/stubs'],
+                        extra_link_args=extra_link_args + [make_relative_rpath('lib')],
+                        )
+    extensions.append(THNVRTC)
```

Comment:
you might need to build with `-Wl,--no-as-needed`. You're not using any of `libnvrtc` or `libcuda` symbols

Reply:
fixed on master